src/pages/publication/contributors_tab/ContributorValidator.tsx

```diff
@@ -1,61 +1,67 @@
-import React, { Dispatch } from 'react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
-import { MenuItem, TextField } from '@material-ui/core';
+import { MenuItem } from '@material-ui/core';
 import PersonIcon from '@material-ui/icons/Person';
 
-import { removeContributor, updateContributor } from '../../../redux/actions/contributorActions';
 import ContributorType from '../../../types/contributor.types';
 import StyledContributor from './StyledContributor';
+import ContributorStyles from './StyledContributor';
+import { Field, useFormikContext, FormikProps } from 'formik';
 
 const StyledContributorValidator = styled(StyledContributor.ContributorContainer)`
-  grid-template-areas: 'icon name institution verify-person verify-person . delete';
+  grid-template-areas: 'icon name institution verify-person verify-person arrows delete';
 `;
 
-const StyledNameInput = styled(TextField)`
+const StyledNameInput = styled(Field)`
   background-color: ${({ theme }) => theme.palette.background.default};
   margin-right: 1rem;
 `;
 
 interface ContributorValidatorProps {
   contributor: ContributorType;
-  dispatch: Dispatch<any>;
+  index: number;
+  remove: (index: number) => void;
+  swap: (indexA: number, indexB: number) => void;
 }
 
-const ContributorValidator: React.FC<ContributorValidatorProps> = ({ contributor, dispatch }) => {
+const ContributorValidator: React.FC<ContributorValidatorProps> = ({ index, remove, swap }) => {
```

Comment:
hva er egentlig forskjellen på denne og contributors? er det to forskjellige komponenter for en forfatter som ikke er verifisert og en som er det?